clang/test/CodeGenSYCL/fp-accuracy.cpp

```diff
@@ -0,0 +1,102 @@
+// RUN: %clang_cc1  -fsycl-is-device -ffp-builtin-accuracy=high:sin,sqrt -ffp-builtin-accuracy=medium:cos -ffp-builtin-accuracy=low:tan -ffp-builtin-accuracy=cuda:exp,acos -ffp-builtin-accuracy=sycl:log,asin -emit-llvm -triple spir64-unknown-unknown -disable-llvm-passes %s -o - | FileCheck %s
```

Comment:
Can you add a run line where a TU value of accuracy is used and a another one with a mix of TU accuracy and function specific ones?

Reply:
Added additional run lines according to your suggestion.